samples/template/README.md

```diff
@@ -0,0 +1,22 @@
+# Savant module template
+
+Start your own project with this template.
+
+## Develop with IDE
+
+Copy the template, rename. Open template with your favorite IDE.
+
+### PyCharm Professional
+1. [Configure a docker interpreter](https://www.jetbrains.com/help/pycharm/using-docker-as-a-remote-interpreter.html) using savant-deepstream container `ghcr.io/insight-platform/savant-deepstream:latest`.
```

Comment:
We should use the Docker image of the module because if the user uses third-party packages specified in the requirements.txt file, they will not be present in the base image.
Therefore, when setting up PyCharm, the user should select the appropriate Docker file and 'context folder', which is the project folder, and choose 'build' instead of 'pull' for the image.
Unfortunately, PyCharm does not automatically collect information about newly installed packages in the Docker container. However, there is an option to manually scan for new packages. To do this, go to the settings and search for "rescan", then navigate to Plugins -> Python -> Rescan Available Python Modules and Packages and set the hotkey. After adding a new package to requirements.txt, simply press the specified hotkey to rebuild the Docker image and update the packages.